cli/src/commands/account.rs

```diff
@@ -40,7 +40,14 @@ impl Account {
             Self::New { seed } => {
                 // Sample a new Aleo account.
                 let account = snarkos_account::Account::from(match seed {
-                    Some(seed) => PrivateKey::<Network>::new(&mut ChaChaRng::seed_from_u64(seed))?,
+                    Some(bigint) => {
+                        let mut bytes = bigint.to_bytes_be().1;
+                        bytes.truncate(32);
```

Comment:
From `resize` [docs](https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.resize) it seems it will handle truncation by itself. However overflowing the 32bytes isn't likely to be what user wanted to achieve so I would drive towards returning `Err` there if the value provided is bigger than 32bytes. Maybe with `anyhow::ensure!`